scripts/make_summary.py

```diff
@@ -471,6 +471,15 @@ def to_csv(dfs):
 
 
 if __name__ == "__main__":
+    if 'snakemake' not in globals():
+        from _helpers import mocksnakemake
+        snakemake = mocksnakemake('make_summary', network='elec', simpl='',
+                           clusters='5', ll='copt', opts='Co2L-24H', country='all')
+        network_dir = os.path.join('..', 'results', 'networks')
```

Comment:
```suggestion
        network_dir = os.path.join('results', 'networks')
```

Comparing with the original code, the `'..'` seem misplaced.

Reply:
no it is actually not, this effect comes again from having paths hard coded in the script. The `mock_snakemake` function expects to be executed in the scripts folder. That's why all paths have to be rearranged in the function. And since this path here doesn't come from the `Snakefile`, it has to be adapted like this

Reply:
I see. Makes sense now.